libraries/azure_generic_resources.rb

```diff
@@ -113,7 +113,9 @@ def api_version_used_for_query_state
   def self.populate_filter_table(raw_data, table_scheme)
     filter_table = FilterTable.create
     table_scheme.each do |col_field|
-      filter_table.register_column(col_field[:column], field: col_field[:field])
```

Comment:
Hi @sathish-progress , could you explain what this change is for?

Reply:
the filter-table library allows us to produce flattened and uniq arrays as columns, as [described](https://github.com/inspec/inspec/blob/master/dev-docs/filtertable-usage.md#a-colors-method-that-will-return-a-flattened-and-uniqd-array-of-values)

and here for some reason, the `style` option was not allowed to be set at individual resources.. Hence the change..